scala/private/phases/phase_write_executable.bzl

```diff
@@ -109,7 +109,7 @@ def _write_executable_non_windows(ctx, executable, rjars, main_class, jvm_flags,
         wrapper.short_path,
     )
 
-    if use_jacoco and _coverage_replacements_provider.is_enabled(ctx):
+    if use_jacoco and ctx.configuration.coverage_enabled:
```

Comment:
Probably unrelated to this PR but what `use_jacoco` stands for? Isn't it covered with `ctx.configuration.coverage_enabled`? I think I saw somewhere a code like this `use_jacoco = ctx.configuration.coverage_enabled`

Reply:
Thanks, let me see if it can be removed